File: c++/RecorridoSecuencial.cpp
#include <iostream>
int main() {
  int numeros[] = {1, 2, 3, 4, 5};
  int tam = sizeof(numeros) / sizeof(numeros[0]);
  for (int i = 0; i < tam; i++) {
    std::cout << numeros[i] << " ";
  }
      return 0;
}

File: java/RecorridoSecuencial.java
public class RecorridoSecuencial {
    public static void main(String [] args){
        int [] numeros = {1, 2, 3, 4, 5};
        String men = "";
        for (int i = 0; i <numeros.length; i++){
            men += numeros[i] + " ";
        }
        System.out.println(men);
    }

}
